### PR TITLE
PYIC 3780: Handle enhanced-verification states in address/fraud subjourney

### PR DESCRIPTION
**Pull request by TalatAydinGDS**
### What changed

added a new separate exitEvent for Fraud's enhanced-verification

### Why did it change

We should probably map it to a separate exit event, and handle properly, or devise a way to handle this better.

### Issue tracking

- [PYIC-3870](https://govukverify.atlassian.net/browse/PYIC-3870)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-3870]: https://govukverify.atlassian.net/browse/PYIC-3870?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TalatAydinGDS** [comment]
deployed to my dev env, conducted some separate tests what same and separate session for f2f Cimit and some happy path test scenarios.

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ ADDRESS_AND_FRAUD:
         next:
           exitEventToEmit: next
         enhanced-verification:
-          exitEventToEmit: next
+          exitEventToEmit: fraudEnhanced
```
Comment (Joe-Edwards-GDS) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml]: Can we just use `enhanced-verification` for this - is there a reason you think we need a new name?

**Review by Joe-Edwards-GDS — COMMENTED** [review]
Looks good, I'd prefer to use a consistent name (`enhanced-verification`) if possible though.

Sorry this was a bit fiddly to explain last week!

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml (hunk 1 above)
Comment (TalatAydinGDS) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml]: I thought, we can follow with separate exitEvent for Fraud. Yeah, you are right. it's not necessary. changed and deployed to my dev env and tested again.

**Review by Joe-Edwards-GDS — APPROVED** [review]

**Pull request merged**